systemtests/scripts/functions

```diff
@@ -499,7 +489,7 @@ change_files()
    # Don't rely on specific filenames and paths,
    # as these might change in the future.
    #
-   DIR=${1:-${BackupDirectory-}}
+   DIR=${1:-${BackupDirectory-"."}}
```

Comment:
Which purpose does this change have?

Reply:
Honestly, I don't remember. BackupDirectory must be set in each individual test. I came across a situation, where it has not be defined. But this must have been during preliminary testing, as the function change_files() is not called by any current systemtests (only in the old regresstests).
So, what to do with this? Remove this change? Or remove the full function?

Reply:
This change is now removed.